apps/communications/contacts/js/contacts.js

```diff
@@ -696,6 +716,10 @@ var Contacts = (function() {
   var enterSearchMode = function enterSearchMode(evt) {
     Contacts.view('Search', function viewLoaded() {
       contacts.List.initSearch(function onInit() {
+        var searchList = document.getElementById('search-list');
```

Comment:
we can have something like

var searchList = ,
      activityName = ,
      activityType =